app/models/bus_stop.rb

```diff
@@ -48,8 +48,22 @@ def route_list
     routes.pluck(:number).sort.join(', ')
   end
 
-  def self.to_csv(limited_attributes: false)
-    # TODO
+  def self.to_csv
+    CSV.generate headers: true do |csv|
+      all_field_names = Field.pluck(:name)
+      stop_attrs = { name: 'Stop Name', hastus_id: 'Hastus ID', route_list: 'Routes', updated_at: 'Last updated' }
+      csv << stop_attrs.values + all_field_names
+      all.each do |stop|
+        stop_data = stop_attrs.keys.map { |attr| stop.send attr }
+        all_field_names.each do |field_name|
+          bsf = BusStopField.find_by(bus_stop: stop, field_name: field_name)
+          if bsf.present?
```

Comment:
Idk why some of these weren't present, but it happened.

Reply:
But you can easily imagine how it might happen in the future. A field is added and then immediately we print a CSV without updating any of the stops, for example.